android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/Activity.kt

```diff
@@ -41,45 +44,35 @@ internal fun Activity.setAsImmersive() {
         window.attributes.layoutInDisplayCutoutMode =
             WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES
     }
+}
 
-    window.getWindowInsetsController().apply {
+private fun hideInsets(insetsController: WindowInsetsControllerCompat) {
+    insetsController.apply {
         hide(WindowInsetsCompat.Type.systemBars())
         systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
     }
 }
 
 /**
- * Anytime a new Window is focused like of a Dialog or of the keyboard the activity
- * will exit immersive mode.
- * This will observe such events and set again the activity to be immersive
- * the next time it gets focused.
+ * Shows the system UI windows that were hidden, thereby exiting the immersive experience.
+ * For Android SDK versions P and above, this function also restores the application's use
```

Comment:
nit: Consistency with reference to the version, e.g. `for devices running API 28+`